src/common/com/intellij/plugins/haxe/ide/highlight/HXMLColorSettingsPage.java

```diff
@@ -33,6 +33,9 @@
   private static final AttributesDescriptor[] DESCRIPTORS = new AttributesDescriptor[]{
     new AttributesDescriptor("Key", HXMLSyntaxHighlighter.KEY),
     new AttributesDescriptor("Value", HXMLSyntaxHighlighter.VALUE),
+    new AttributesDescriptor("Comment", HXMLSyntaxHighlighter.COMMENT),
+    new AttributesDescriptor("Include", HXMLSyntaxHighlighter.INCLUDE),
```

Comment:
you able to include other hxml files into your main hxml file. It just works because hxml file is just haxe compiler arguments format, so one of arguments can be other hxml file